src/Centreon/Domain/Repository/InformationsRepository.php

```diff
@@ -71,9 +71,14 @@ public function authorizeMaster(string $ip): void
         $sql = "DELETE FROM `informations` WHERE `key` = 'authorizedMaster'";
         $stmt = $this->db->prepare($sql);
         $stmt->execute();
+
+        /*
+         * resolve the address down to IP
+         */
+        $ipAddress = gethostbyname($ip);
```

Comment:
maybe this preparation should be done before IP is pass to the repository method